src/test/java/uk/nhs/tis/trainee/usermanagement/api/UserAccountResourceTest.java

```diff
@@ -74,12 +76,16 @@ void setUp() {
 
   @Test
   void shouldReturnExistenceFalseWhenUserAccountNotExists() throws Exception {
-    UserAccountDetailsDto userAccountDetails = new UserAccountDetailsDto(USERNAME, "NO_ACCOUNT",
-        "NO_ACCOUNT", List.of(), null);
+    UserAccountDetailsDto userAccountDetails = UserAccountDetailsDto.builder()
+        .id(ID)
+        .email(EMAIL)
```

Comment:
It probably doesn't matter, but the service would actually return just the mfaStatus and userStatus fields, if I'm reading it correctly.